sdk/storage/storage-blob/src/policies/AnonymousCredentialPolicy.ts

```diff
@@ -10,6 +10,9 @@ import { CredentialPolicy } from "./CredentialPolicy";
  * or for use with Shared Access Signatures (SAS).
  */
 export class AnonymousCredentialPolicy extends CredentialPolicy {
+  /* eslint-disable @typescript-eslint/no-useless-constructor */
+  // This constructor doesn't have more things to do except initialize its super class,
+  // still need this constructor to initialize AnonymousCredentialPolicy instances.
```

Comment:
I recommend updating the comment to something that explains the underlying issue. Please consider the below suggestion and apply it to other policies as well

```suggestion
  // The base class has a protected constructor. Adding a public one to enable constructing of this class.
```